include/glow/Optimizer/GraphOptimizer/CompilationContext.h

```diff
@@ -49,6 +49,16 @@ struct PrecisionConfiguration {
 
 using QuantizationMode = PrecisionConfiguration::QuantizationMode;
 
+/// Options relevant to optimizations during compilation.
+struct OptimizationOptions {
+  /// If true, perform graph optimizations.
+  bool enableGraphOptz{true};
```

Comment:
Is this ever set to false? Is it still necessary?

Reply:
Not for this PR. But could be useful in general. I'll remove it from this PR to keep it focused. These flags probably belong more into the PassManager anyways ;-)